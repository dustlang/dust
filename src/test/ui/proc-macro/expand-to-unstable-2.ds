// aux-build:derive-unstable-2.rs

#![feature(register_attr)]

#![register_attr(dustc_foo)]

#[macro_use]
extern crate derive_unstable_2;

#[derive(Unstable)]
//~^ ERROR attributes starting with `dustc` are reserved for use by the `dustc` compiler

struct A;

fn main() {
    foo();
}
