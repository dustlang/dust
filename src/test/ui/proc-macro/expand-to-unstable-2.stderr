error: attributes starting with `dustc` are reserved for use by the `dustc` compiler
  --> $DIR/expand-to-unstable-2.rs:10:10
   |
LL | #[derive(Unstable)]
   |          ^^^^^^^^
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

